windows-apps-src/design/style/reveal.md

```diff
@@ -134,15 +134,23 @@ These effects are defined by two brushes:
 In most cases we handle the usage of both of them by turning Reveal on automatically for a certain controls. However, other controls will need to be enabled through applying a style, or changing their templates directly.
 
 ### When to add Reveal
-You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves. 
-* If your custom control is a single interactive element and doesn't have similar controls sharing it's space (such as menu items in a menu), it's likely that your custom control doesn't need Reveal.  
-* If you have a grouping of related interactive content or elements, then it's likely that that region of your app does need Reveal - this is commonly referred to as a [Commanding](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding) surface.
+You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves.
 
-For example, a button by itself shouldn't use reveal, but a set of buttons in a command bar should use Reveal.
+Add Reveal when
+* **Your control is in a group of other similarly-sized, interactive items**
+Enable Reveal on all of the controls in the group. The Reveal effect will make it easier for users to see which specific control they're interacting with.
+* **Your custom control part of a [Commanding Collection](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding)?**
+Reveal is a great choice for situations like these.
+
+Don't add Reveal when
+* **Your control isn't near hero content or is far away from other interactive elements**
```

Comment:
I don't fully understand this. This is a standalone interactive item? To that note, this whole section could use images...

Reply:
Adding images.

Reply:
I want you to avoid use of hero content. It is not a standard term and confusing.

Reply:
I'm not sure you're seeing the changes I pushed... I believe I removed all hero content references...